students/ZhuojingXie/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+
+
+#series 1
+list1 =  ["Apples", "Pears", "Oranges" , "Peaches"]
+print(list1)
+fruit1 = input("Tell me a fruit you like > ")
+list1.append(fruit1)
+print(list1)
+num1 = int(input("Now, tell me a numer you like between 1 to {} > ".format(len(list1))))
+while num1<0 or num1 > len(list1):
+    num1 = int(input("Please pick up a number between 1 to {} ".format(len(list1))))
+print("Okay, seems the number you choose are {}, and the fruit corresponding to that number is {}".format(num1, list1[num1-1]))
+fruit1 = input("Tell me another fruit you like> ")
+list1 = [fruit1] + list1
+print("The list now is : ", list1)
+fruit1 = input("Okay, now the last fruit you would like to choose> ")
+list1.insert(0,fruit1)
+print("The list now is : ", list1)
+print("the fruit stating with letter 'P' is ")
+for fruit in list1:
+    if fruit[0] == 'P' or fruit[0] == 'p':
```

Comment:
you can use 

`fruit[0].lower() == 'p'`

or

`fruit.lower().startswith('p')`